### ISSUE TITLE
[Feature Request] Expose properties to make the control Accessible

### ISSUE THREAD

**Issue by shanranm** [lilcodelab/Xamarin.Plugin.Calendar#108]
**Is your feature request related to a problem? Please describe.**
As a developer, I have little to no control to add Accessibility tags on different views of the Plugin

**Describe the solution you'd like**
Exposing an `AccessibilityText` property and setting it to `AutomationProperties.HelpText` would be helpful

**Describe alternatives you've considered**
No alternatives. Since the accessibility needs to go on each day baking it into the plugin would be an awesome addition!

**Additional context**
A couple of more scenarios to think of - properties to customize the text for selected/disabled days/months/years
